lib/interferon/destinations/datadog.rb

```diff
@@ -349,6 +353,7 @@ def self.same_alerts(alert, people, alert_api_json)
         query: alert_api_json['query'].strip,
         message: alert_api_json['message'].strip,
         evaluation_delay: alert_api_json['options']['evaluation_delay'],
+        new_host_delay: alert_api_json['options']['new_host_delay'],
```

Comment:
The Datadog API returns `300` (default value) for all alerts. If we do the check here, then we'll also being updating existing alerts because our default value is `nil`.

I think the solution would be to change the default value within `AlertDSL` to `300`